etc/layout_viewer/viewer.html

```diff
@@ -100,22 +100,23 @@ <h1> Servo Layout Viewer </h1>
             }
 
             function create_flow_tree(trace_node) {
-                var base = get_base(trace_node);
-
                 var node = {
-                    text: trace_node.class + " (" + base.id + ")",
-                    id: base.id,
-                    icon: "dummy",
-                    href: "#diff-" + base.id
+                    text: Object.keys(trace_node)[0],
+//                    id: base.id,
+//                    icon: "dummy",
+//                    href: "#diff-" + base.id
```

Comment:
Here and in three other places, prefer removing code rather than leaving it commented out. Git history still has it.

Reply:
Yes, this was uncommented on a later commit. Sorry about that